01 - Navigation/01 - Navigate to something you know/1.1. Project Tool Window.txt

```diff
@@ -1,10 +1,10 @@
 Project Tool Window
 
-Shows all files in project.
+Show all files in project.
```

Comment:
Please read the rest of the _original_ files. I just rephrased a few to have all written in the same way. Who or what the described thing does is up to the reader, I'd say.